### ISSUE TITLE
Just curious why does the accuracy for the permuted MINST is so low

### ISSUE THREAD

**Issue by shixianwen** [moskomule/ewc.pytorch#1]
Hi:
I also want to implement the EWC in pytorch by myself.
But according to the Counterpart Tensorflow implementation:
https://github.com/ariseff/overcoming-catastrophic/blob/master/experiment.ipynb
their accuracy is so high.
Why does the Pytorch implementation has only 0.7 in your implementation.
Many thanks
Shixian Wen

**Comment by moskomule** [comment]
I'm not sure... Maybe because the original paper and your reference use training accuracy but I use test accuracy...? Or of course, my implementation may include some mistakes.
